tuna.go

```diff
@@ -146,6 +146,8 @@ type Common struct {
 
 	reverseBytesExitToEntry map[string][]uint64
 	reverseBytesEntryToExit map[string][]uint64
+
+	minBalance float64 // minimum wallet balance requirement for connecting
```

Comment:
Use `Fixed64` type and parse it with `StringToFixed64` method

Reply:
Yes, modified.